salt/modules/boto_elb.py

```diff
@@ -159,31 +185,20 @@ def create(name, availability_zones, listeners=None, subnets=None,
 
     CLI example to create an ELB::
 
-        salt myminion boto_elb.create myelb '["us-east-1a", "us-east-1e"]' listeners='[[443, 80, "HTTPS", "HTTP", "arn:aws:iam::1111111:server-certificate/mycert"]]' region=us-east-1
+        salt myminion boto_elb.create myelb '["us-east-1a", "us-east-1e"]' listeners='{"elb_port": 443, "elb_protocol": "HTTPS", ...}' region=us-east-1
     '''
     conn = _get_conn(region=region, key=key, keyid=keyid, profile=profile)
-
-    if __salt__['boto_elb.exists'](name, region, key, keyid, profile):
+    if exists(name, region, key, keyid, profile):
         return True
     if isinstance(availability_zones, string_types):
         availability_zones = json.loads(availability_zones)
     if isinstance(listeners, string_types):
         listeners = json.loads(listeners)
-    # Combining listeners and complex_listeners together makes our lives
-    # easier in some ways, especially since during introspection you can
-    # only get a combined set of listeners back from boto; however, boto
-    # requires us to send in separate listeners and complex listeners and
-    # the only real difference is the size. It feels like amazon/boto hate
-    # developers and wish to make us suffer.
-    _listeners = []
     _complex_listeners = []
     for listener in listeners:
```

Comment:
Good catch. I'd argue it doesn't make sense to create an ELB without at least one listener (and I don't think AWS supports this) so we should remove the default `None` in the function signature. How does that sound (vs. adding an `is None` check in the body of the function)?